### PR TITLE
[batch] test batch directly

### PR DESCRIPTION
**Pull request by cseed**

### CODE REVIEW

**Comment by cseed** [comment]
Hmm, still broken, will investigate.

File: batch/test/serverthread.py [hunk 1]
```diff
@@ -8,17 +8,17 @@
...
         def ping():
             return Response(status=200)
 
-        self.host = host or os.environ.get('POD_IP', '127.0.0.1')
-        self.port = port
+        self.host = os.environ['HAIL_BATCH_WORKER_IP']
+        self.port = os.environ['HAIL_BATCH_WORKER_PORT']
         self.app = app
-        self.server = make_server(self.host, self.port, app)
+        self.server = make_server('0.0.0.0', 5000, app)
```
Comment (danking) [inline batch/test/serverthread.py]: this port should be `self.port`, no?

Comment (cseed) [reply batch/test/serverthread.py]: No.  self.port is the port docker binds on the host, it is forwarded to port 5000 on the container, which is what we request in the `port:` field of test_batch step in build.yaml.

**Comment by cseed** [comment]
bump.  This would be nice to have for monitoring batch tests.

**Pull request merged**